app/controllers/api/v1/surveys_controller.rb

```diff
@@ -33,6 +43,15 @@ def create
 
   # PATCH/PUT /surveys/1 or /surveys/1.json
   def update
+    @survey.update(survey_params)
```

Comment:
Are we going to allow live surveys to be updated?

Reply:
good catch